examples/trace-analytics-sample-app/docker-compose.yml

```diff
@@ -3,10 +3,8 @@ services:
   data-prepper:
     restart: unless-stopped
     container_name: data-prepper
-    image: amazon/opendistro-for-elasticsearch-data-prepper:latest
-    command: sh data-prepper-wait-for-odfe-and-start.sh
+    image: opensearchproject/data-prepper:latest
     volumes:
-      - ./resources/data-prepper-wait-for-odfe-and-start.sh:/usr/share/data-prepper/data-prepper-wait-for-odfe-and-start.sh
```

Comment:
It looks like this script would start Data Prepper. I think we might need two other changes:

1. Add Data Prepper to the Docker Compose file
2. Remove this script as well.

Reply:
I took another look. The Docker image is going to start data-prepper without the command. So I think item 1 above was incorrect.

All we need then is to remove that old script since it should no longer be in use.

Reply:
Good point! The unused "data-prepper-wait-for-odfe-and-start.sh" has been removed.